packages/jest-reporters/src/verbose_reporter.ts

```diff
@@ -118,14 +118,14 @@ export default class VerboseReporter extends DefaultReporter {
           if (test.status === 'pending') {
             result.pending += 1;
           } else if (test.status === 'todo') {
-            result.todo += 1;
+            result.todo.push(test);
           } else {
             this._logTest(test, indentLevel);
           }
 
           return result;
         },
-        {pending: 0, todo: 0},
+        {pending: 0, todo: [] as Array<TestResult.AssertionResult>},
```

Comment:
I think it might be nice to type this slightly differently. Instead of using `as X` inline you can tell the `reduce` what it is folding into with `array.reduce<Type>(fn);`

```ts
  const summedTests = tests.reduce<{ pending: Number, todo: Array<TestResult.AssertionResult>}>(
    (result, test) => {
      if (test.status === 'pending') {
        result.pending += 1;
      } else if (test.status === 'todo') {
        result.todo.push(test);
      } else {
        this._logTest(test, indentLevel);
      }

      return result;
    },
    {pending: 0, todo: [] },
  );
```

Reply:
Sure let me make the changes in abit